frontend/src/components/inputs/EntityForm.vue

```diff
@@ -110,6 +129,10 @@ export default {
     batchMode: {
       type: Boolean,
       default: false
+    },
+    attributes: {
```

Comment:
If I read teh code in EntityBulkAdd.vue correctly, an entity data object is passed in as `attributes` property. Would it make sense to reuse the `entity` property instead?

Reply:
Yes, same as above -> https://github.com/SUSE/aimaas/pull/201#discussion_r1574443807